PKHeX.Core/Saves/Substructures/Gen3/SecretBase3.cs

```diff
@@ -59,7 +61,7 @@ public ushort SecretBasesReceived
     public Span<byte> GetDecorationCoordinates() => Data.Slice(0x22, 0x10);
     public void SetDecorationCoordinates(Span<byte> value) => value.CopyTo(GetDecorationCoordinates());
 
-    private Span<byte> TeamData => Data.Slice(50, 72);
+    private Span<byte> TeamData => Data.Slice(0x34, 108);
```

Comment:
What rationale for 108 bytes? The party data struct seems to be 0x48 bytes large:
https://github.com/pret/pokeemerald/blob/18f84b78f2d1a8669753fa586836fca06036c790/include/global.h#L538-L546